docs/ca/dev/email-service/email-deliverability.md

```diff
@@ -0,0 +1,84 @@
+If the server and domain has been reported before for spamming or distributing malicious content, it can be that the message that it wants to transmit will either be outright rejected or flagged as spam.
+There are several domain reputation checkers available that you can use to assess the reputation of your email domain.
+To improve your domain's reputation and increase the deliverability of your emails, there are several things you should do:
+
+#### Debug and troubleshoot carefully
+Your Spryker PaaS non-production environments are - by default - using a sandboxed Simple Email Service Account. This means that you can only send emails to [validated recipients](https://docs.spryker.com/docs/ca/dev/email-service/verify-email-addresses.html).
+This protects you from mistakenly sending faulty emails to many recipients that might report this behavior and damage the reputation of your sender's domain.
+Only request the SES Sandbox to be disabled if you are confident that your email functionality works as expected
+
+#### Configure SPF, DKIM and DMARC
+There are extensive resources available on these three concepts, so we will only provide a short explanation. We will be using example.com as an example email domain.
+
+**SPF (Sender Policy Framework)**
+Purpose: Lists authorized servers for sending emails from a domain.
+How It Works: Mail servers check SPF records before delivering emails.
+How to configure:
+When you are using Spryker PaaS, you are using AWS SES to send emails. To entitle this email server to send emails with your domain, you will need to set
+the following DNS records (TXT) for the domain you are using to send emails:
+```bash
+v=spf1 include:amazonses.com -all
+```
+
+**DKIM (DomainKeys Identified Mail)**
+Purpose: Digitally signs emails to verify their origin.
+How It Works: Public key cryptography ensures authenticity.
+How to configure:
+In the AWS Console, switch the the SES Dashboard. You will be able to find the DKIM DNS CNAMES in the DKIM section under Configuration>Identities.
+You should find three DNS CNAME records in this format. You will need to set them as CNAMES for the email domain you are using.
+
+```bash
+CNAME
+NAME 123EXAMPLEHASH123._domainkey.example.com
+VALUE 456EXAMPLEHASH456.dkim.amazonses.com
+
+CNAME
+NAME 789EXAMPLEHASH789._domainkey.example.com
+VALUE 789EXAMPLEHASH789.dkim.amazonses.com
+
+CNAME
+NAME abcEXAMPLEHASHabc._domainkey.example.com
+VALUE abcEXAMPLEHASHabc.dkim.amazonses.com
+```  
+
+**DMARC (Domain-based Message Authentication Reporting and Conformance)**
+Purpose: Handles emails that fail SPF or DKIM checks.
+How It Works: Specifies actions (e.g., quarantine, reject) based on authentication results.
+How to configure:
+In the AWS Console, switch the the SES Dashboard. You will be able to find the DMARC DNS entries in the DMARC section under Configuration>Identities.
+You will need to set an MX and TXT record
```

Comment:
```suggestion
You will need to set an MX and TXT records:
```

Reply:
I tried to follow these instructions, and failed to find how to setup MX/TXT records.
Also, from the text below it's not clear what's the value for MX record.

Reply:
https://docs.aws.amazon.com/ses/latest/dg/receiving-email-mx-record.html this page suggests that MX record is controlled in the DNS.